blog/2022-03-02-os.mdx

```diff
@@ -1,14 +1,15 @@
 ---
 slug: os-iasql
-title: IaSQL is now open source
+title: SELECT 'Hello World!' FROM iasql;
 authors: [depombo, dfellis, aguillenv]
 ---
 
-We are excited to announce that IaSQL is now open source. The main repository is under https://github.com/iasql/iasql-engine! As perfectionists, we feel like IaSQL will never be truly ready. However, we believe IaSQL is at the point where it can start to be useful for developers managing infrastructure in the cloud. IaSQL is a SaaS that lets you model your infrastructure as data by maintaining a 2-way connection between your AWS account and a Postgres SQL database to represent the definitive state (and status) of your cloud which cannot be achived with a static infrastructure declaration. This means that when you connect an AWS account to an IaSQL instance it automatically backfills the database with your existing cloud resources.
+We are excited to announce that [IaSQL](https://iasql.com) is now open source. The main repository is under https://github.com/iasql/iasql-engine! As perfectionists, we feel like IaSQL will never be truly ready. However, we believe IaSQL is at the point where it can start to be useful for developers managing infrastructure in the cloud. IaSQL is a SaaS that lets you model your infrastructure as data by maintaining a 2-way connection between your AWS account and a Postgres SQL database to represent the definitive state (and status) of your cloud which cannot be achieved with a static infrastructure declaration. This means that when you connect an AWS account to an IaSQL instance it automatically backfills the database with your existing cloud resources. No need to painstakingly redefine, or reconcile, your existing infrastructure, and IaSQL's [module system](https://docs.iasql.com/module/) means you can specify which parts of your cloud infrastructure you wish to control.
```

Comment:
```suggestion
We are excited to announce that [IaSQL](https://iasql.com) is now open source. The main repository is under https://github.com/iasql/iasql-engine! As perfectionists, we feel like IaSQL will never be truly ready. However, we believe IaSQL is at the point where it can start to be useful for developers managing infrastructure in the cloud. IaSQL is a SaaS that lets you model your infrastructure as data by maintaining a 2-way connection between your AWS account and a Postgres SQL database to represent the definitive state (and status) of your cloud which cannot be achieved with a static infrastructure declaration. This means that when you connect an AWS account to an IaSQL instance it automatically backfills the database with your existing cloud resources. No need to painstakingly redefine or reconcile your existing infrastructure, and IaSQL's [module system](https://docs.iasql.com/module/) means you can specify which parts of your cloud infrastructure you wish to control.
```

These two commas are unnecessary and without them there the comma joining the second half of the paragraph makes it clearer that there's a subject change (to me).